website/source/docs/commands/taint.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "docs"
+page_title: "Command: taint"
+sidebar_current: "docs-commands-taint"
+description: |-
+  The `terraform taint` command manually marks a Terraform-managed resource as tainted, forcing it to be destroyed and recreated on the next apply.
+---
+
+# Command: taint
+
+The `terraform taint` command manually marks a Terraform-managed resource
+as tainted, forcing it to be destroyed and recreated on the next apply.
+
+This command _will not_ modify infrastructure, but does modify the
+state file in order to mark a resource as tainted. Once a resource is
+marked as tainted, the next
+[plan](/docs/commands/plan.html) will show that the resource will
+be destroyed and recreated and the next
+[apply](/docs/commands/apply.html) will implement this change.
+
+Forcing the recreation of a resource is useful when you want a certain
+side effect of recreation that is not visible in the attributes of a resource.
+For example: re-running provisioners will cause the node to be different
+or rebooting the machine from a base image will cause new startup scripts
+to run.
+
+Note that tainting a resource for recreation may affect resources that
+depend on the newly tainted resource. For example, a DNS resource that
+uses the IP address of a server may need to be modified to reflect
+the potentially new IP address of a tainted server. The
+[plan command](/docs/commands/plan.html) will show this if this is
+the case.
+
+## Usage
+
+Usage: `terraform taint [options] name`
+
+The `name` argument is the name of the resource to mark as tainted.
+The format of this argument is `TYPE.NAME`, such as `aws_instance.foo`.
```

Comment:
I bet users would appreciate something like a `terraform list` command to enumerate resources in a given config in this format.